README.md

```diff
@@ -1,4 +1,6 @@
 # hello-world
 This is my first repository.
 
-This is my first commit.
\ No newline at end of file
+This is my first commit.
+
+Welcome to Github Sleepy!
```

Comment:
Please change the "Sleepy" word to "Álmos"!

Reply:
Please change the "Sleepy" word to "Álmos"!

Reply:
@savolyialmos Done by 4c5437dc99e8129ef3e8b9dff30462a52a3239b commit.